starter-code/models/celebrity.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+let celebSchema = new mongoose.Schema({
+    name: String, 
+    occupation: String,
+    catchPhrase: String
+})
+
+let celebModel = mongoose.model('model', celebSchema);
```

Comment:
'model' will be the name of your model and collection. So to be more descriptive of the content, it would be better to use something like 'celebrity' similar to how you did on the other model